R/mapvizieR_object.R

```diff
@@ -49,6 +49,17 @@ mapvizieR.default <- function(raw_cdf, raw_roster) {
   )
   
   class(mapviz) <- "mapvizieR"
+  
```

Comment:
any thoughts on why the object becomes class `mapvizieR` before the accelerated goals?